pyseer/input.py

```diff
@@ -586,15 +586,22 @@ def iter_variants(p, m, cov, var_type, burden, burden_regions, infile,
         if eof:
             return
 
+        v = p.values
+        c = cov.values
+        if k is not None:
+            pattern = hash_pattern(k)
+        else:
+            pattern = None
+        
         if (k is None) or not (min_af <= af <= max_af) or (missing > max_missing):
-            yield (None, None, None, None, None, None,
-                   None, None, None, None, None, None,
-                   None, None, None, None)
+            # TODO: currently the way to know if a variant was
+            # af filtered is to check that phenotypes vector is None
+            # a more robust way could be better
+            yield (var_name, None, k, m, c, af, pattern,
+                   lineage_effects, lineage_clusters,
+                   filter_pvalue, lrt_pvalue, null_fit, firth_null,
+                   kstrains, nkstrains, continuous)
```

Comment:
As mentioned in the comment relying on the phenotypes vector to be empty to recognize pre-filtered variants could be a potential problem. Can leave it like this for now and reimplement the way variants to be ignored are handled by each regression method

Reply:
I suppose we could pass another boolean around, but this seems ok to me for now with the comment to remind us

Reply:
Sure!